open_alchemy/__init__.py

```diff
@@ -8,6 +8,7 @@
 from sqlalchemy.ext import declarative
 
 from open_alchemy import types as oa_types
+from open_alchemy.build import PackageFormat
```

Comment:
The pattern I have been following is to import at least the module. This makes mocking possible if required for testing. It also helps with namespaces. E.g. change this line to `from open_alchemy import build` and then refer to it using `build. PackageFormat`.